### ISSUE TITLE
Allow this.props.onChange for API calls

### ISSUE THREAD

**Issue by checkhavoc** [prakhar1989/react-tags#18]
Any chance this can be added to the input onChange handler? This would be nice to have so that the input can make API requests just in case suggestions are not already pre-loaded. If there is a better way to handle this type of request, please let me know. Thanks!

```javascript
handleChange: function handleChange(e) {

        // Check if this.props.onChange exists
        if (this.props.onChange){
            this.props.onChange(e.target.value.trim())
        }
        var query = e.target.value.trim();
        var suggestions = this.props.suggestions.filter(function (item) {
            return item.toLowerCase().search(query.toLowerCase()) === 0;
        });

        this.setState({
            query: query,
            suggestions: suggestions
        });
    },
```

Jayr Camacho

**Comment by prakhar1989** [comment]
Fixed in #19

**Comment by atley** [comment]
I dont get it. I have a Search API for tags. I want to update the suggestion Array onChange. How do I do that?
